.github/scripts/updateIssueTemplateComponents.js

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env node
+const fs = require("fs");
+const path = require("path");
+
+const componentsDir = path.join(process.cwd(), "packages", "src", "components");
+const componentBlacklist = new Set(["avoid-me", "dont-add-me-to-list", "should-not-be-included"]);
```

Comment:
How will these predefined denied dirs be used?

Reply:
Where are the exclusions coming from?

Reply:
Gotcha. We could keep it within the script for now, but we might be able to define that elsewhere (e.g, component/doc metadata).